lib/block-ref-grammar.ym

```diff
@@ -61,7 +62,8 @@ block_arg
           LL_BLOCK
           {
             cfg_lexer_pop_context(lexer);
-            cfg_args_set(*result, $1, $3); free($1); free($3);
+            gchar *normalized_key = __normalize_key($1);
+            cfg_args_set(*result, normalized_key, $3); free($1); free($3); free(normalized_key);
           }
```

Comment:
normalized_key should be freed by g_free() while tokens should be freed by simple free()